Makefile

```diff
@@ -394,3 +394,13 @@ else
 		-r $(REPO_OWNER)/$(REPO_NAME) \
 		-Z > codecov-upload.log
 endif
+
+## -- Target for maintaining consistent crd copies --
+
+.PHONY: consistent-crds
+## Copy crd from manifests/ and cr from examples/nodejs_postgresql to deploy/crds/
+consistent-crds:
+	$(Q)cd ./deploy/crds/ && ln -srf ../../manifests-upstream/${OPERATOR_VERSION}/servicebindingrequests.apps.openshift.io.crd.yaml \
+	apps_v1alpha1_servicebindingrequest_crd.yaml
+	$(Q)cd ./deploy/crds && ln -srf ../../examples/nodejs_postgresql/service-binding-request.yaml \
+	apps_v1alpha1_servicebindingrequest_cr.yaml
```

Comment:
Why are crd and cr being copied from manifests and example to deploy/crds? Aren't, crds and cr from deploy/crds suppose to be original and we create manifests from it?